android-app/app/src/androidTest/java/de/dhbw/ka/se/fibo/ui/adding/ItemDialogAddingFragmentTest.java

```diff
@@ -0,0 +1,178 @@
+package de.dhbw.ka.se.fibo.ui.adding;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.replaceText;
+import static androidx.test.espresso.action.ViewActions.scrollTo;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withChild;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.not;
+import static de.dhbw.ka.se.fibo.TestMatchers.hasTextInputLayoutErrorText;
+
+import org.junit.Test;
+
+import de.dhbw.ka.se.fibo.R;
+
+public class ItemDialogAddingFragmentTest extends AddingFragmentTest {
+    //Item Tests
+
+    private final String firstItemAmount = "1.0";
+    private final String expectedFirstItemAmount = "1.0 Stk.";
+    private final String firstItemPrice = "1";
+    private final String expectedFirstItemPrice = "1,00 €";
+    private final String firstItemName = "Erster Beispielartikel";
+    private final String editedFirstItemAmount = "1.5";
+    private final String expectedEditedFirstItemAmount = "1.5 Stk.";
+    private final String editedFirstItemPrice = "2.3123123";
+    private final String expectedEditedFirstItemPrice = "2,31 €";
+    private final String editedFirstItemName = "Bearbeiteter Beispielartikel";
+
+    @Test
+    public void testSuccessfulItemAdding() {
+        onView(withId(R.id.add_item_button)).perform(scrollTo(), click());
+
+        //interact with dialog
+        onView(withId(R.id.adding_fragment_dialog_container)).check(matches(isDisplayed()));
```

Comment:
```suggestion
        onView(withId(R.id.adding_fragment_dialog_container))
                     .check(matches(isDisplayed()));
```
for a better overview